### ISSUE TITLE
Extracting ROMs from Xbox 360 XBLA

### ISSUE THREAD

**Issue by rwebster85** [farmerbb/RED-Project#14]
I've written up a guide (with a link to a Reddit post with a few original steps taken from it) on extracting ROMs, like Sega Classics XBLA games, from your Xbox 360. No modding or "hacking" required. Just a USB stick and some programs on your PC.

https://www.retrogameboards.com/t/the-ripping-thread-how-to-build-your-own-legit-retro-rom-library/98/168?u=rwebster

**Comment by rwebster85** [comment]
@farmerbb, I'm trying to extract more ROMs from Xbox Live, and I'm having issues finding information about extracting .m files from Sega Vintage collection.

![image](https://user-images.githubusercontent.com/16655270/110239258-f2501800-7f3d-11eb-9e0c-aea9d67907d4.png)

I've tried using mdfTools (mdfDecompressor.exe) which works for other Sega ROMs in other collections, but no joy with these ones.

Since these files we made by m2, and have a .m extension (but they have mxb in the header when viewed in a HEX editor) similar to your Castlevania and Mega Drive mini tutorials, I'm wondering if your MArchiveBatchTool file will be able to decompress/extract the ROM from these. I don't know enough about them to know which commands/seeds etc to use though.

What do you think?

**Comment by Dragoonglue** [comment]
Have you tried using [this version of mdfTools](https://www.mediafire.com/file/s6qm0faay8bk6zh/mdfTools.rar/file)? It worked for me with the files from the Switch rerelease of Alex Kidd.

**Comment by rwebster85** [comment]
@Dragoonglue Yes mate that's the same one I used. Were the Switch files also .m, with "mxb" in the rom header?

**Comment by eisnerguy1** [comment]
I'm trying to extract the roms from the Monster World Vintage Collection and I'm having the same issue. The files also have an mxb in the header.

**Comment by charlieglide** [comment]
A lot of arcade games can be easily extracted by simply using WxPirs.  These are often labeled as a rom.  I've been able to extract "Tron", "Discs of Tron", "Time Pilot", "Scramble", "Yie-Ar Kung Fu", "Track & Field", "Frogger", "Super Contra".  Open them in a hex editor and copy and paste the relevant data to a new file and rename to the correct file MAME specifies. 

Some collections require the use of a program called binwalk and the use of the command binwalk -e <name of file in directory>  

These games include Shinobi, Altered Beast and Golden Axe.  You need to deinterleave and the cpu data does not match, but the games will run.  The delisted games The Simpsons and X-Men will work with this method as well.

**Comment by mjBrickman** [comment]
I figured out a way to extract Alex Kidd in Miracle World from "Sega Vintage Collection: Alex Kidd & Co." Here is what I did.
I ran the game in Xenia, an Xbox 360 emulator, and opened HxD, a hex editor that has the ability to read things running in RAM. Using documentation of Master System ROMs from [here](https://www.smspower.org/Development/ROMHeader), I was able to locate the position of "TMR SEGA" in the RAM and using this info,  I was able to find the ROM itself. While I may have not figured out the MXB format of the psb.m, I found a way to bypass needing to understand the compression format.

**Comment by RealRelativeEase** [comment]
> These games include Shinobi, Altered Beast and Golden Axe. You need to deinterleave and the cpu data does not match, but the games will run. The delisted games The Simpsons and X-Men will work with this method as well.

Are there any detailed instructions or a tutorial on how to do this? I have some of those games and would be interested in getting playable ROMs of those titles.
